<launch>
    <node name="listener" pkg = "exosystem" type = "listener.py"  launch-prefix="xterm -e" respawn="true"/>
    <node name="encoder_talker" pkg = "exosystem" type = "Encoder_read.py" launch-prefix="xterm -e" required="true"/>
    <node name="motorcontrolabd" pkg = "exosystem" type = "motorcontrolabd" launch-prefix="xterm -e" required="true"/>
    <node name="force_talker" pkg = "exosystem" type = "Force_Data_Read.py" launch-prefix="xterm -e" required="true"/>
</launch>